redhat-operators/automation-broker/automationbroker.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: automationbroker
```

Comment:
The package name for the operator needs to be unique from the upstream version. We can make the community version `automationbroker-community` and leave this one be. Same goes for templateservicebroker